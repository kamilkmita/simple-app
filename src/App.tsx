import MainPage from "./modules/mainPage/MainPage";
import "./App.css";

const App = () => <MainPage />;

export default App;
