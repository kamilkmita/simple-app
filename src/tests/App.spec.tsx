import renderer from "react-test-renderer";
import App from "../App";

jest.mock("../modules/mainPage/itemList/ItemList");

describe("App", () => {
  test("proper render", () => {
    const component = renderer.create(<App />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
