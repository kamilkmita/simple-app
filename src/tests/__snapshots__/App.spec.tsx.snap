// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App proper render 1`] = `
<div
  className="MuiContainer-root MuiContainer-fixed MuiContainer-maxWidthLg"
>
  <h1
    className="sc-bdvvaa"
  >
    Premier league clubs
  </h1>
  ItemListMockedComponent
</div>
`;
